src/main/java/lotto/model/LottoTicket.java

```diff
@@ -4,7 +4,7 @@
 
 public class LottoTicket {
 
-    public static final Long PRICE = 1_000L;
+    public static final int PRICE = 1_000;
     public static final int SIZE = 6;
 
     private final SortedSet<LottoNumber> lottoNumbers;
```

Comment:
가급적 하위 구현체인 SortedSet 보다는 아래와 같이 상위의 인터페이스 타입인 Set로 타입을 선언하시는 것이 좋습니다. 

```suggestion
        private final Set<LottoNumber> lottoNumbers;
```

[[OOP] 우리는 왜 List list = new ArrayList(); 라고 쓰는가](https://www.itsmo.dev/interface-as-type/) 글을 참고해보시면 도움이될 것 같아요.

Reply:
반영하였습니다.